source/prerequisites.rst

```diff
@@ -92,6 +93,25 @@ Here is a configuration example for ``Nginx`` web server using ``php-fpm``.
         }
     }
 
+lighttpd configuration
+^^^^^^^^^^^^^^^^^^^^^^
+
+Here is a virtual host configuration example for ``lighttpd`` web server.
+
+.. warning::
+   The following configuration is only suitable for GLPI version 10.0.7 or later.
+
+.. code-block:: lighttpd
+
+    $HTTP["host"] =~ "glpi.localhost" {
+        server.document-root = "/var/www/glpi/public/"
+
+        url.rewrite = (
+            "^(.*)$" => "/index.php/$1"
+        )
```

Comment:
Recommended improvement to let lighttpd serve the static files:
`url.rewrite-if-not-file = ("" => "/index.php${url.path}${qsa}")`